server/websocket.go

```diff
@@ -585,19 +585,6 @@ func (p *Plugin) handleLeave(us *session, userID, connID, channelID string) erro
 		p.LogDebug("timeout waiting for reconnection", "userID", userID, "connID", connID, "channelID", channelID)
 	}
 
-	state, err := p.getCallState(channelID, false)
-	if err != nil {
-		return err
-	}
```

Comment:
This is actually significant from a perf perspective since fetching the state is one of the heaviest operations we make.